src/routes/safe/components/Transactions/TxsTable/columns.tsx

```diff
@@ -65,22 +67,26 @@ export const getTxAmount = (tx: Transaction, formatted = true): string => {
     return NOT_AVAILABLE
   }
 
+  if (!symbol) {
```

Comment:
lets get rid of getting token symbol information there and use our component for that

Reply:
The component is for displaying the symbol, this call is used to format the amount